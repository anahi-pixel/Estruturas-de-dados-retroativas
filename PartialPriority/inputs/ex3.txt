1 1 100
1 2 200
1 3 300
1 4 400
5
6
3 1
5
6
