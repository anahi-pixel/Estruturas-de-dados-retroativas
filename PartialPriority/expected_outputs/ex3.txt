100
4
200
3
